build/script.sh

```diff
@@ -0,0 +1,36 @@
+# Install nasm
+sudo apt-get install nasm
+
+# Install a 32-bit environment if a 32-bit arch is used
+[ -z $1 ] || sudo dpkg --add-architecture i386 && \
+             sudo apt-get update && \
+             sudo apt-get install gcc-multilib g++-multilib \
+                                  libpython3.6-dev:i386
+
+# Install Cython
+sudo pip3 install cython
+
+# Install zimg
+git clone --depth 1 https://github.com/sekrit-twc/zimg.git
+cd zimg
+./autogen.sh
+./configure $1 $2 $3 $4
```

Comment:
Instead of passing the arguments from within the CI yaml, I'd pass one argument (like "i686"), and then set the appropriate arguments to `./configure` from within the script, like this:
```sh
if [ "$1" = "i686" ]; then
    CONFIGURE_ARGS="CFLAGS=-m32 and so on"
else
    CONFIGURE_ARGS=""
fi
# ...
./configure $CONFIGURE_ARGS
```
This way if changes to the building are required they would be isolated to the actual build script where they belong.

Reply:
You're perfectly right, I've used your approach for the `Windows` script but not with this one. Thanks for spotting the error! :)